

class InsertException(Exception):

    def __init__(msg="", error=None):

        pass